#include <iostream>
#include <vector>

using namespace std;
int main()
{
	vector<int> num{ 1,2,3,4,5 };

	try {
		num.at(10);
	}
	catch (...) {
		cout << "Out of bound\n";
	}
	return 0;
}
